seed/lib/mappings/data/pm-mapping.json

```diff
@@ -269,6 +422,15 @@
     "type": "string",
     "schema": ""
   },
+  {
+    "display_name": "PM Property ID",
+    "to_field": "pm_property_id",
+    "to_table_name": "PropertyState",
+    "from_field": "Portfolio Manager ID",
```

Comment:
💯  can't believe we didn't have this before -- just a note, looks like it's only included when importing from data exported as "Download your entire portfolio"